lib/internal/Magento/Framework/Model/AbstractExtensibleModel.php

```diff
@@ -128,8 +128,8 @@ protected function initializeCustomAttributes()
                     $customAttributes[$customAttributeCode] = $customAttribute;
```

Comment:
Are you sure that these changes need to be added?

Reply:
sry , that is another issue commits was mixed in this , i have forced commit.